app/CMakeLists.txt

```diff
@@ -89,41 +89,30 @@ target_sources(
 
 include(AndroidNdkModules)
 android_ndk_import_module_native_app_glue()
-target_link_libraries(native-lib native_app_glue)
+target_link_libraries(native-lib PRIVATE native_app_glue)
 set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -u ANativeActivity_onCreate")
 endif()
 
-
 if(OPENXR)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DXR_USE_PLATFORM_ANDROID -DXR_USE_GRAPHICS_API_OPENGL_ES")
-    include_directories(
-            ${CMAKE_SOURCE_DIR}/../third_party/OpenXR-SDK/include
-            ${CMAKE_SOURCE_DIR}/../app/src/openxr/cpp
-    )
-    if (OCULUSVR)
-        include_directories(
-            ${CMAKE_SOURCE_DIR}/../third_party/OVRPlatformSDK/Include
-        )
-        add_custom_command(TARGET native-lib POST_BUILD
-                COMMAND ${CMAKE_COMMAND} -E copy
-                ${CMAKE_SOURCE_DIR}/../third_party/ovr_openxr_mobile_sdk/OpenXR/Libs/Android/${ANDROID_ABI}/Release/libopenxr_loader.so
-                ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/libopenxr_loader.so
-                )
-    elseif (HVR)
+    find_package(OpenXR REQUIRED CONFIG)
+    include_directories(${CMAKE_SOURCE_DIR}/../app/src/openxr/cpp)
+    if (HVR)
+        include_directories(${CMAKE_SOURCE_DIR}/../third_party/OpenXR-SDK/include)
         add_custom_command(TARGET native-lib POST_BUILD
                 COMMAND ${CMAKE_COMMAND} -E copy
                 ${CMAKE_SOURCE_DIR}/../third_party/hvr/${ANDROID_ABI}/libxr_loader.so
                 ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/libxr_loader.so
                 )
     elseif (PICOXR)
-        include_directories(
-            ${CMAKE_SOURCE_DIR}/../third_party/picoxr/include
-        )
+        include_directories(${CMAKE_SOURCE_DIR}/../third_party/OpenXR-SDK/include)
```

Comment:
FYI, you should be able to do this instead:

```
target_link_libraries(native-lib PRIVATE OpenXR::headers)
```

to just use the headers from the AAR